### ISSUE TITLE
Say hello to kevin

### ISSUE THREAD

**Issue by kevinhowbrook** [davidecatani/portfolio#3]
:wave: hello !!

**Comment by imshuffling** [comment]
Whats going on Davide???

**Comment by davidecatani** [comment]
Hey @imshuffling  and @kevinhowbrook , just finally  taking care of my github account and preparing it for future opportunities. 
I'm studying Next.js and I'm taking the opportunity to build my portfolio with that, but I'm just beginning it now.

What about you guys? how things are going?

**Comment by imshuffling** [comment]
All good here :) been using react for quite a few years, ive dabbled with next.js but mainly used gatsby. I think Next.js is a safe bet to learn

**Comment by davidecatani** [comment]
Yeah, I've been using Angular for something like 3/4 years but it's useless if you're looking for a job outside Italy so I'm updating my skills :)

**Comment by imshuffling** [comment]
Why has this been closed? :)

**Comment by davidecatani** [comment]
Exactly, I didn't approve it.
